src/components/Pagination.jsx

```
@@ -1,35 +1,74 @@
+
 export default function Pagination({ page, setPage, totalPages }) {
+  if (totalPages === 0) return null;
+
   const handlePrev = () => setPage(Math.max(1, page - 1));
   const handleNext = () => setPage(Math.min(totalPages, page + 1));
 
+  const getPageNumbers = () => {
+    const pages = [];
+    const maxVisible = 5;
+    let startPage = Math.max(1, page - 2);
+    let endPage = Math.min(totalPages, startPage + maxVisible - 1);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'endPage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)